src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1358,9 +1358,13 @@ public enum Mode
                 /// </summary>
                 EndShiftReconnections,
                 /// <summary>
-                /// End and start control connections.
+                /// Begin duplicate connection.
                 /// </summary>
-                EndAndStartCtrlConnection,
+                BeginDuplicateConnection,
```

Comment:
oye... public breaking change. can you keep the old enum, obsolete it, and add the new ones at the end without changing the order?

Reply:
@mjkkirschner Can I reuse the same name for the new enum?  Or do you mean use the `[Obsolete]` tag on the actual enum items?  I don't think the first is possible